src/main.rs

```diff
@@ -213,6 +214,109 @@ struct Args {
         env = "OTEL_EXPORTER_OTLP_ENDPOINT"
     )]
     otel_exporter_otlp_endpoint: Option<String>,
+    #[clap(long = "otel-metrics-endpoint", env = "OTEL_EXPORTER_METRICS_ENDPOINT", default_value = metrics::DEFAULT_OTEL_METRICS_ENDPOINT)]
+    otel_metrics_endpoint: String,
+    #[clap(long = "otel-metrics-mode", env = "OTEL_EXPORTER_METRICS_MODE", default_value_t = OtlpMode::Http)]
+    otel_metrics_mode: OtlpMode,
+    // Enable metrics
+    #[clap(
+        long = "enable-metrics",
+        default_value_t = false,
+        env = "WASMCLOUD_ENABLE_METRICS"
```

Comment:
It's a big change but what do you think about calling this `metrics-enabled`? That would enable you to use `WASMCLOUD_METRICS_ENABLED` which makes `WASMCLOUD_METRICS_` as a prefix consistent. 

That said, I also get wanting to keep the UX of `--enable-<x>`... I personally like it when tools have all their options related to a certain feature/area consistently ENV-var'd. I know it's not a thing we've been doing up until now (otel options are just out there "in space", so to speak), but wanted to note it.